bundles/core/src/main/java/com/adobe/cq/commerce/core/components/internal/models/v1/product/ProductImpl.java

```diff
@@ -395,4 +396,18 @@ private ProductInterface getProduct(GraphqlClient client, String slug) {
 
         return null;
     }
+
+    private String safeDescription(ProductInterface product) {
+        ComplexTextValue description = product.getDescription();
+        if (description == null) {
+            return "";
+        }
+
+        String descriptionHtml = description.getHtml();
+        if (descriptionHtml == null) {
+            return "";
```

Comment:
Why return an empty string? The HTL should handle that correctly, simply testing for the property before displaying anything.

Reply:
That's probably the easier approach. So I let the function just return `null`.

Reply:
@cjelger done :)